Formula/abyss.rb

```diff
@@ -0,0 +1,32 @@
+class Abyss < Formula
+  desc "Genome sequence assembler for short reads"
+  homepage "http://www.bcgsc.ca/platform/bioinfo/software/abyss"
+  url "https://github.com/bcgsc/abyss/releases/download/2.0.2/abyss-2.0.2.tar.gz"
+  sha256 "d87b76edeac3a6fb48f24a1d63f243d8278a324c9a5eb29027b640f7089422df"
+
+  option "with-openmp", "Enable OpenMP multithreading"
+
+  needs :openmp if build.with? "openmp"
+
+  depends_on "boost" => :build
+  depends_on "google-sparsehash" => :build
+  depends_on :mpi => :cc
+
+  def install
+    system "./autogen.sh" if build.head?
+
+    args = [
```

Comment:
These arguments should be inlined since nothing is optionally appended to the array.

Reply:
But, do we need to use `--disable-openmp` when building without it?

Reply:
It checks for openmp and disables it since clang doesn't support it. I've added it in regardless.